src/quo2/components/list_items/preview_list/view.cljs

```diff
@@ -0,0 +1,181 @@
+(ns quo2.components.list-items.preview-list.view
+  (:require [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.markdown.text :as quo2.text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]
+            [react-native.hole-view :as hole-view]))
+
+(def params
```

Comment:
hmm, params is a bit confusing as that's more like parameters of a function imo, i.e something we expect to be passed in. I would consider more like properties or something like that 🤔